### PR TITLE
[CLOUD-878] minor fixes and links from README

### PR DESCRIPTION
**Pull request by spron-in**
[CLOUD-878](https://jira.percona.com/browse/CLOUD-878) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

minor fixes and links to website and forum

[CLOUD-878]: https://perconadev.atlassian.net/browse/CLOUD-878?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| custom-extensions | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/5d56cf1d7/custom-extensions.log) |
| custom-tls | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/5d56cf1d7/custom-tls.log) |
| demand-backup | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/5d56cf1d7/demand-backup.log) |
| finalizers | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/5d56cf1d7/finalizers.log) |
| init-deploy | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/5d56cf1d7/init-deploy.log) |
| monitoring | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/5d56cf1d7/monitoring.log) |
| one-pod | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/5d56cf1d7/one-pod.log) |
| operator-self-healing | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/5d56cf1d7/operator-self-healing.log) |
| pitr | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/5d56cf1d7/pitr.log) |
| scaling | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/5d56cf1d7/scaling.log) |
| scheduled-backup | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/5d56cf1d7/scheduled-backup.log) |
| self-healing | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/5d56cf1d7/self-healing.log) |
| start-from-backup | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/5d56cf1d7/start-from-backup.log) |
| tablespaces | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/5d56cf1d7/tablespaces.log) |
| telemetry-transfer | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/5d56cf1d7/telemetry-transfer.log) |
| upgrade-consistency | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/5d56cf1d7/upgrade-consistency.log) |
| upgrade-minor | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/5d56cf1d7/upgrade-minor.log) |
| users | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/5d56cf1d7/users.log) |
| We run 18 out of 18|

commit: https://github.com/percona/percona-postgresql-operator/pull/975/commits/5d56cf1d7f94b9aa76530d9cf7f5192ad646aa41
image: `perconalab/percona-postgresql-operator:PR-975-5d56cf1d7`

File: README.md [hunk 1]
```diff
@@ -55,35 +55,18 @@ kubectl apply --server-side -f https://raw.githubusercontent.com/percona/percona
 kubectl apply -f https://raw.githubusercontent.com/percona/percona-postgresql-operator/main/deploy/cr.yaml
 ```
 
-## Contributing
+# Need help?
 
-Percona welcomes and encourages community contributions to help improve Percona Operator for PostgreSQL.
+**Commercial Support**  | **Community Support** |
+:-: | :-: |
+| <br/>Enterprise-grade assistance for your mission-critical MongoDB deployments with the Percona Operator for MongoDB. Get expert guidance for complex tasks like multi-cloud replication, database migration and building platforms.<br/><br/>  | <br/>Connect with our engineers and fellow users for general questions, troubleshooting, and sharing feedback and ideas.<br/><br/>  |
```
Comment (hors) [inline README.md:62]: PG?

**Review by hors — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (spron-in) [reply README.md]: Fixed! Thank you

**Review by fiowro — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| custom-extensions | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/f5fcf9da1/custom-extensions.log) |
| custom-tls | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/f5fcf9da1/custom-tls.log) |
| demand-backup | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/f5fcf9da1/demand-backup.log) |
| finalizers | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/f5fcf9da1/finalizers.log) |
| init-deploy | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/f5fcf9da1/init-deploy.log) |
| monitoring | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/f5fcf9da1/monitoring.log) |
| one-pod | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/f5fcf9da1/one-pod.log) |
| operator-self-healing | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/f5fcf9da1/operator-self-healing.log) |
| pitr | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/f5fcf9da1/pitr.log) |
| scaling | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/f5fcf9da1/scaling.log) |
| scheduled-backup | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/f5fcf9da1/scheduled-backup.log) |
| self-healing | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/f5fcf9da1/self-healing.log) |
| start-from-backup | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/f5fcf9da1/start-from-backup.log) |
| tablespaces | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/f5fcf9da1/tablespaces.log) |
| telemetry-transfer | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/f5fcf9da1/telemetry-transfer.log) |
| upgrade-consistency | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/f5fcf9da1/upgrade-consistency.log) |
| upgrade-minor | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/f5fcf9da1/upgrade-minor.log) |
| users | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pg-operator/PR-975/f5fcf9da1/users.log) |
| We run 18 out of 18|

commit: https://github.com/percona/percona-postgresql-operator/pull/975/commits/f5fcf9da1cdad698faa9b42df846323468973743
image: `perconalab/percona-postgresql-operator:PR-975-f5fcf9da1`

**Pull request merged**